script/check-code-style.js

```diff
@@ -119,7 +119,7 @@ tests().forEach(file => {
 });
 
 const isIgnoredForInterfaceCheck = (file) => {
-  return ['applicationmetadata'].includes(file);
+  return ['applicationmetadata', 'statscollector'].includes(file);
```

Comment:
In `check-code-style.js` script, we check if a file is an interface by checking if the name of the file is equal to the directory name.
For example:
Directory name: `src/statscollector`
Origin interface: `src/statscollector/StatsCollector.ts`

This is not 100% right today, so I need to manually exclude `statscollector`.

Reply:
Just wonder what would fail if you do not do this? If it is just commenting then I think we should document the public API methods in the class itself

Reply:
Yes, just need to add comments for all APIs.
It's just weird that a class is treated as an interface, it confused me for a while.
But it's not a big deal. I will add comments and remove this change here.